cmd/dump.go

```diff
@@ -82,10 +86,14 @@ func runDump(ctx *cli.Context) error {
 	reposDump := path.Join(tmpWorkDir, "gitea-repo.zip")
```

Comment:
reposDump could be in below `else`?

Reply:
Yes, if we reorder a bit the instructions.

Reply:
Done